src/main/java/dev/slimevr/gui/AutoBoneWindow.java

```diff
@@ -79,12 +80,14 @@ public void run() {
 										try {
 											if (poseRecorder.isReadyToRecord()) {
 												setText("Recording...");
-												// 1000 samples at 20 ms per sample is 20 seconds
+												// 1000 samples at 20 ms per sample is
+												// 20 seconds
 												int sampleCount = server.config.getInt("autobone.sampleCount", 1000);
 												long sampleRate = server.config.getLong("autobone.sampleRateMs", 20L);
-												Future<PoseFrames> framesFuture = poseRecorder.startFrameRecording(sampleCount, sampleRate);
+												Future<PoseFrames> framesFuture = poseRecorder
+														.startFrameRecording(sampleCount, sampleRate);
```

Comment:
Really not a fan of the double-indent when lines are broken up for wrapping. Can we change it to single indent? Unless I'm missing something I don't see the reason for it. Its a lot more clear in this tab-indented file why it is an extravagant use of whitespace (the files prior to this are two-space indented)

Reply:
Update: Yes its totally possible to control the indent level, I confirmed this in the eclipse settings:
![image](https://user-images.githubusercontent.com/6969415/167352352-14f4bacb-549c-4808-a834-b82024a3d47e.png)

Reply:
I know you can, idk if it looks better though with single indent.